backend/src/main/java/com/crm/api/v1/AuthenticationController.java

```diff
@@ -33,14 +45,23 @@
         return ResponseEntity.ok(new JwtResponse(jwtTokenUtils.generateToken(userDetails), jwtTokenUtils.generateRefreshToken(userDetails)));
     }
 
+    @ApiOperation(value = "Create and save new Account in database")
+    @PostMapping("/newAccount")
```

Comment:
Better convention will be "/new-account"

Reply:
Proponuje tutaj dać /register